src/content/docs/projects/ToPWR/Communication Channels.md

```diff
@@ -0,0 +1,90 @@
+2. **Dokumentuj ustalenia**: Po zakończeniu ważnej dyskusji, kluczowe wnioski przenieś do odpowiedniego miejsca (np. do issue na GitHub albo Google Docs).
+3. **Używaj wątków**: Pamiętaj o używaniu wątków do dłuższych rozmów (threads).
+
+### Messenger
+
+**Cel**:
+
+- Szybkie informacje lub ogólne ogłoszenia.
+- Sytuacje wymagające błyskawicznej reakcji (ASAP).
+- „Miękkie” sprawy organizacyjne (dogadywanie terminów spotkań, potwierdzenia, krótkie pytania).
+
+**Zalecenia**:
+
+1. **Proste pytania i szybkie decyzje**: Jeżeli potrzebujesz natychmiastowej odpowiedzi — Messenger jest najlepszy.
+2. **Nie nadużywaj**: Częste powiadomienia na Messengerze mogą rozpraszać. Jeśli sprawa nie jest pilna, rozważ użycie Discord.
+3. **Potwierdzaj kluczowe ustalenia**: Po zakończeniu rozmowy w Messengerze, która ma wpływ na projekt, zapisz krótkie podsumowanie np. w notatce lub w innym, bardziej „trwałym” kanale (np. Discord/Google Docs). Możesz też odezwać się do jednego z kierowników projektu.
+
+### GitHub (Komentarze/Pull Requesty)
+
+**Cel**:
+
+- Dyskusje i uwagi na temat konkretnych fragmentów kodu.
+- Omawianie pull requestów i dokumentowanie zmian w kodzie.
+- Zgłaszanie i rozwiązywanie issues.
+
+**Zalecenia**:
+
+1. **Konkretne komentarze**: Komentuj bezpośrednio w kodzie przy danym fragmencie (np. w PR), aby wiadomo było, do czego odnosisz swoją uwagę.
+2. **Opisowe PR**: Twórz pull requesty z opisem kontekstu, co i dlaczego zostało zmienione oraz jak przetestować te zmiany.
+3. **Używaj szablonów**: Jeśli repozytorium twojego projektu posiada szablony PR lub issue, wypełniaj je, aby zachować spójny format w całym zespole.
+
+## Sugerowane dobre praktyki i dodatkowe pomysły dla kierowników
+
+1. **Automatyczne integracje**:
+
+   - Rozważ włączenie powiadomień z GitHuba do dedykowanego kanału na Discordzie. Dzięki temu zespół może na bieżąco widzieć nowe PR i dyskusje.
+   - Jeśli macie system zarządzania zadaniami (np. GitHub Proejcts, Asanę, Trello, Notion, Jirę), warto ustawić integrację z Discordem, by kluczowe akcje (np. zamknięcie zadania) były widoczne od razu.
+
+2. **Porządkowanie wątków**:
+
+   - W Messengerze trudno wrócić do starych wątków. Jeśli temat wymaga dłuższych ustaleń, **przenieś** dyskusję na Discord.
+   - W przypadku krytycznych decyzji dot. kodu — przechodź na GitHuba (Issues/PR), aby był łatwy dostęp do kontekstu w przyszłości.
+
+3. **Etykieta komunikacyjna**:
+
+   - Unikaj pisania w godzinach nocnych lub w trakcie ustalonych okresów, gdy projekt ma przerwę, jeśli nie jest to konieczne. Zwłaszcza na Messengerze, gdzie powiadomienia przychodzą bezpośrednio na telefon (oraz często na konta prywante, a nie dedykowane do pracy).
+
+4. **Archiwizacja**:
+   - Po zakończeniu projektu lub sprintu, przejrzyjcie kanały (szczególnie na Discordzie i Messengerze), by najważniejsze informacje trafiły do oficjalnej dokumentacji (np. w Handbooku, repozytorium, lub Google Drive).
```

Comment:
Może to nie wchodzi w tematykę posta, dotyczącej narzędzi komunikacjii, ale skoro jest poruszony wątek dla kierowników, który ma charakater ogólny to brakuje mi wyszczególnienia roli google drive'a. ( jeśli chodzi o ToPWR )  ALe może to jest temat na osobnego handbooka, odnoście dokumentacji projektu itp.

Reply:
Chyba, że wszystko z google docs, będziemy przenosić na solvrodocsy

Reply:
słuszna uwaga, ale to chyba rzeczywiście do innego guide damy, a tu tylko wspomnimy 

tutaj damy odnośniki do tego nowego